blist/models.py

```diff
@@ -21,9 +22,13 @@ def __init__(self, **kwargs):
         self.vanity_url: str = kwargs.get("vanity_url")
         self.server_count: int = kwargs.get("server_count")
         self.shard_count: int = kwargs.get("shard_count")
-        self.add_date: datetime.datetime = datetime.datetime.fromtimestamp(kwargs.get("add_date"))
+        self.joined: datetime.datetime = datetime.datetime.fromisoformat(kwargs.get("joined").strip("Z"))
```

Comment:
```suggestion
        self.add_date: datetime.datetime = datetime.datetime.fromisoformat(kwargs.get("joined").strip("Z"))
```
We shouldn't be breaking backward compatibility for our current users, though there are probably no current users except for you.